src/components/SearchBar.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./SearchBar.css";
+
+export default function SearchForm({
+  onHandleFormSubmit,
+  onHandleSearchInputChange
+}) {
+  return (
+    <form onSubmit={onHandleFormSubmit}>
+      <input
+        type='text'
+        name='city'
+        onChange={onHandleSearchInputChange}
+        placeholder='Search City...'
+        style={{ margin: "10px", padding: "5px" }}
```

Comment:
You already have a SearchBar css file, you could put al your styling in the css to keep it in one place.